backend/src/v2/driver/driver.go

```diff
@@ -1645,8 +1646,8 @@ func getDAGTasks(
 			// handled with dsl.Collected.
 			_, ok := v.GetExecution().GetCustomProperties()["iteration_count"]
 			if ok {
-				glog.Infof("Found a ParallelFor task, %v. Skipping it.", v.TaskName())
-				continue
+				glog.V(4).Infof("Found a ParallelFor task, %v.", v.TaskName())
+				parallelFor = true
```

Comment:
Nit: It's hard to read code when a function argument value is overridden. Please consider using a different variable name.